ts/sagas/wallet.ts

```diff
@@ -882,10 +905,7 @@ function* continueWithPaymentMethodsHandler(
     yield call(checkPayment, pagoPaClient, paymentId);
   }
 
-  // in case  (paymentId === undefined && !hasPaymentId),
-  // the payment id could not be fetched successfully. Handle
-  // the error here @https://www.pivotaltracker.com/story/show/159400682
-
+  // redirect as needed
```

Comment:
not sure here, perhaps you need a return on error above, it looks like this point gets reached also in case of paymentErrors...

Reply:
Absolutely, you are right. Fixed it some lines above this